backend/de.metas.adempiere.adempiere/base/src/main/java/de/metas/bpartner/service/BPPrintFormatQuery.java

```diff
@@ -37,36 +41,36 @@
  *
  */
 
+@Getter
 @Value
 @Builder
 public class BPPrintFormatQuery
 {
-	@Getter
-	private final BPartnerId bpartnerId;
+	BPartnerId bpartnerId;
+
+	@Nullable
+	DocTypeId docTypeId;
 
-	@Getter
-	private final int docTypeId;
+	@Nullable
+	AdTableId adTableId;
 
-	@Getter
-	private final int adTableId;
+	int printFormatId;
```

Comment:
since you replaced adTableId, docTypeId, etc with their Object ID, 

why not using `PrintFormatId` here too?

Reply:
thx, fixed